packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js

```diff
@@ -491,6 +491,23 @@ QUnit.module('decorators markup', {}, () => {
         assert.notStrictEqual(window.getComputedStyle($deleteToggleIcon).backgroundImage, 'none', 'background image is defined');
     });
 
+    QUnit.test('list item aria-label should be equal to item text, if showSelectionControls is true and selectionMode is multiple (T1248422)', function(assert) {
```

Comment:
Should it really be applied only in this case?

Reply:
[c19e53a751cd9eb55c8fa738214d2017f4c5b39c, c3eb095cdae339944cf3b7594bc501cb64c0185a]